CardParts/src/Classes/Card Parts/CardPartVideoView.swift

```diff
@@ -17,10 +17,16 @@ public class CardPartVideoView: UIView, CardPartView {
     public var videoUrl:URL
     
     // player instance
-    fileprivate var player:AVPlayer!
+    fileprivate var player: AVPlayer!
```

Comment:
nit: Can we expose this also as public, this instance might be required if we wanna customize player controls like volume, pause etc..

Reply:
@badrinathvm this can already be accessed through `(viewController as? AVPlayerViewController)?.player` to customize. I could make it `private(set) public var player` if you think it needs to be more easily accessible, but probably best to keep it from being set publicly.

Reply:
`(viewController as? AVPlayerViewController)?.player` is good enough. Approved.